pallas-traverse/src/header.rs

```diff
@@ -45,4 +46,57 @@ impl<'b> MultiEraHeader<'b> {
             MultiEraHeader::Byron(x) => x.to_hash(),
         }
     }
+
+    pub fn leader_vrf_output(&self) -> Result<ByteVec, Error> {
```

Comment:
Could we make the fn name just `leader_vrf`? Or do you think that the `output` prefix provides more context?

Reply:
A VRF has two parts. There's the output and the proof. In this case, we only are extending to get the output part of it. If you call it something other than "output" in your code, we can maybe change it.

Reply:
makes sense now.